x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -167,7 +168,7 @@ export interface TriggersAndActionsUIPublicPluginStart {
 interface PluginsSetup {
   management: ManagementSetup;
   home?: HomePublicPluginSetup;
-  cloud?: { isCloudEnabled: boolean };
+  cloud: CloudSetup;
```

Comment:
I think the type should be optional, correct?

Reply:
You're right, thank you!